code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -47,3 +47,32 @@
 	embedding = list(embed_chance=100, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
 	wound_falloff_tile = -5
 	shrapnel_type = /obj/item/ammo_casing/harpoon
+
+// Rebar (Rebar Crossbow)
+/obj/projectile/bullet/rebar
+	name = "rebar"
+	icon_state = "rebar"
+	damage = 30
+	speed = 0.4
+	dismemberment = 1 //because a 1 in 100 chance to just blow someones arm off is enough to be cool but also not enough to be reliable
+	armour_penetration = 10
+	wound_bonus = -20
+	bare_wound_bonus = 20
+	embedding = list(embed_chance=60, fall_chance=2, jostle_chance=2, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=3, jostle_pain_mult=2, rip_time=10)
+	embed_falloff_tile = -5
+	wound_falloff_tile = -2
+	shrapnel_type = /obj/item/stack/rods
+
+/obj/projectile/bullet/rebarsyndie
```

Comment:
You resolved the last review of me suggesting this be a subtype saying you would, but didn't

Reply:
I had to change it back, as the way the mag worked meant engi bows would be able to fire syndie ammo which I didn't want to be possible. The syndie bows having better ammo is the main reason I made it a tot item.

Reply:
Is it okay if I mark this as resolved?